src/main/java/io/cdap/delta/bigquery/BigQueryUtils.java

```diff
@@ -33,39 +33,29 @@
 import io.cdap.cdap.api.data.schema.Schema;
 import io.cdap.delta.api.DMLEvent;
 import io.cdap.delta.api.SourceTable;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 import java.util.Set;
 import java.util.UUID;
-import java.util.regex.Pattern;
 import javax.annotation.Nullable;
 
 /**
  * Utility class for executing queries on BigQuery.
  */
 public final class BigQueryUtils {
-  private static final Logger LOG = LoggerFactory.getLogger(BigQueryUtils.class);
   public static final int FIELD_NAME_MAX_LENGTH = 128;
   static final String BACKTICK = "`";
   private static final int DATASET_OR_TABLE_NAME_MAX_LENGTH = 1024;
-  // Valid BigQuery dataset names can contain only letters, numbers, and underscores.
-  // See here: https://cloud.google.com/bigquery/docs/datasets#dataset-naming
-  private static final Pattern VALID_DATASET_NAME_REGEX = Pattern.compile("[\\w]+");
-  private static final Pattern INVALID_DATASET_NAME_REGEX = Pattern.compile("[^\\w]+");
-  // Valid BigQuery table names can contain only Unicode characters in category L (letter), M (mark), N (number),
-  // Pc (connector, including underscore), Pd (dash), Zs (space).
-  // See here: https://cloud.google.com/bigquery/docs/tables#table_naming
-  private static final Pattern VALID_TABLE_NAME_REGEX = Pattern.compile("[\\p{L}\\p{M}\\p{N}\\p{Pc}\\p{Pd}\\p{Zs}]+");
-  private static final Pattern INVALID_TABLE_NAME_REGEX =
-          Pattern.compile("[^\\p{L}\\p{M}\\p{N}\\p{Pc}\\p{Pd}\\p{Zs}]+");
-  private static final String BIG_QUERY_DUPLICATE_ERROR = "duplicate";
+  // according to big query dataset and table naming convention, valid name should only contain letters (upper or
+  // lower case), numbers, and underscores
+  private static final String VALID_NAME_REGEX = "[\\w]+";
```

Comment:
Why was this change required? As per https://cloud.google.com/bigquery/docs/tables#table_naming, table names can contain unicode characters as well, I don't think `[\\w]+` will handle unicode characters except `[A-Za-z0-9_]`